File: backend/DotNet/ClassLibrary/UserDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary
{
    public class UserDTO
    {
        public required Guid Id { get; set; }
        public required string Name { get; set; }
        public required string Phone { get; set; }
        public required bool Ban { get; set; }
        public required UserRole UserRole { get; set; }
        public Guid? Author { get; set; }
        public bool IsBannable { get; set; }
    }
}

File: mobile/employee/app/src/main/java/com/example/g_bankforemployees/feature/users_list/data/model/UserDto.kt
package com.example.g_bankforemployees.feature.users_list.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class UserDto(
    val id: String,
    val name: String? = null,
    val phone: String? = null,
    val ban: Boolean,
    @SerialName("userRole") val userRole: String,
    val author: String? = null,
    val isBannable: Boolean = false,
)
